src/blocked.c

```diff
@@ -210,11 +210,11 @@ void unblockClient(client *c, int queue_for_reprocessing) {
     }
 
     /* We count blocked client stats on regular clients and not on module clients */
-    if (!(c->flags & CLIENT_MODULE)) server.blocked_clients--;
+    if (!(c->flag.module)) server.blocked_clients--;
```

Comment:
Can we remove the parenthesis in the remaining cases? They should look like a normal bool value
```suggestion
    if (!c->flag.module) server.blocked_clients--;
```